aws-organizations-account/src/main/java/software/amazon/organizations/account/UpdateHandler.java

```diff
@@ -98,12 +100,19 @@ else if ((previousParentIds == null && parentIds == null) || previousParentIds.e
                                 .makeServiceCall(this::moveAccount)
                                 .handleError((organizationsRequest, e, proxyClient1, model1, context) -> {
                                     if (e instanceof DuplicateAccountException) {
-                                        log.log(String.format("Got %s when calling %s for "
+                                        logger.log(String.format("Got %s when calling %s for "
                                                         + "account id [%s], source id [%s], destination id [%s]. Continue with next step.",
                                                 e.getClass().getName(), organizationsRequest.getClass().getName(), model.getAccountId(), sourceId, destinationId));
                                         return ProgressEvent.progress(model1, context);
+                                    } else if (e instanceof SourceParentNotFoundException) {
+                                        logger.log("Thrown SourceParentNotFoundException when calling MoveAccount - translating to InvalidInputException.");
```

Comment:
add policyId info here, otherwise we cannot tell which policyId throws exception. We had this issue before, see the exception but did not know who throws it during log diving.